ddl/column.go

```diff
@@ -242,6 +242,11 @@ func onDropColumn(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 		colInfo.State = model.StateWriteOnly
 		// Set this column's offset to the last and reset all following columns' offsets.
 		adjustColumnInfoInDropColumn(tblInfo, colInfo.Offset)
+		err = checkTableInfoValid(tblInfo)
```

Comment:
we should check if the tableInfo valid before we overwrite the tableInfo in TiKV.

Reply:
done. PTAL

Reply:
We have added `check` on line 265, why do it again?